.travis.yml

```diff
@@ -21,74 +20,87 @@ jobs:
     include:
 
         - stage: Test
-          env: SYMFONY_VERSION='4.0.*'
+          name: Symfony 4.0
+          before_install:
+              - *Xdebug
+              - composer require --dev --no-update symfony/symfony=4.0.*
           php: 7.2
 
         - stage: Test
-          env: SYMFONY_VERSION='3.4.*'
+          name: Symfony 3.4
+          before_install:
+              - *Xdebug
+              - composer require --dev --no-update symfony/symfony=3.4.*
           php: 7.2
 
         - stage: Test
-          env: DEPENDENCIES=low
+          name: Lowest dependencies
```

Comment:
Could do with a better name. 'Lowest dependency versions'?

Reply:
If there were another dependencies variation, it could be `Dependencies: lowest` keeping `Dependencies` as a prefix for quick comparison. But I don't see one so no opinion on what reads best.